bundle/deploy/lock/release.go

```diff
@@ -17,11 +18,12 @@ const (
 )
 
 type release struct {
-	goal Goal
+	goal           Goal
+	successMessage string
 }
 
-func Release(goal Goal) bundle.Mutator {
-	return &release{goal}
+func Release(goal Goal, successMessage string) bundle.Mutator {
+	return &release{goal, successMessage}
```

Comment:
This ideally should be in different mutator which can be sequenced with other mutators

Reply:
Indeed, simplified!